dotCMS/src/main/java/com/dotcms/rest/api/v1/system/role/RoleLayoutForm.java

```diff
@@ -0,0 +1,52 @@
+package com.dotcms.rest.api.v1.system.role;
+
+import com.dotcms.rest.api.Validated;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import java.util.Set;
+
+@JsonDeserialize(builder = RoleLayoutForm.Builder.class)
+public class RoleLayoutForm extends Validated {
+
+    private final String roleId;
+    private final Set<String> layoutIds;
+
+    private RoleLayoutForm(final RoleLayoutForm.Builder builder) {
+        roleId    = builder.roleId;
+        layoutIds = builder.layoutIds;
+        checkValid();
+    }
+
+    public String getRoleId() {
+        return this.roleId;
+    }
+
+    public Set<String> getLayoutIds() {
+        return this.layoutIds;
+    }
+
+    public static final class Builder {
+
+        @JsonProperty(required = true)
+        private String roleId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field roleId has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)